src/components/NcGuestContent/NcGuestContent.vue

```diff
@@ -66,7 +72,7 @@ export default {
 </style>
 
 <style lang="scss">
```

Comment:
Why, does scoped not work here?

Reply:
The #content div is outside of the component (in other words: the parent of the component). Using `scoped` and ::v-deep only applies to copmonents that are one ore more levels below the current one.

Reply:
Ah okay, I see